modules/gdscript/gdscript_editor.cpp

```diff
@@ -2173,7 +2173,7 @@ static void _find_identifiers(const GDScriptCompletionContext &p_context, bool p
 
 	static const char *_type_names[Variant::VARIANT_MAX] = {
 		"null", "bool", "int", "float", "String", "Vector2", "Rect2", "Vector3", "Transform2D", "Plane", "Quat", "AABB", "Basis", "Transform",
-		"Color", "NodePath", "RID", "Object", "Dictionary", "Array", "PackedByteArray", "PackedIntArray", "PackedRealArray", "PackedStringArray",
+		"Color", "NodePath", "RID", "Object", "Dictionary", "Array", "PackedByteArray", "PackedInt32Array", "PackedFloat32Array", "PackedStringArray",
```

Comment:
I think the versions exposed to GDScript should be the 64-bit versions, since `int` in GDScript is always 64-bit and `float` in GDScript is always 64-bit, an array of them should also be 64-bit. The 32-bit versions should just be for internal use.

Also, can we keep the exposed name without the size, for simplicity? `int` isn't `int64` in GDScript, etc. So `PackedInt64Array` would appear to users as `PackedIntArray` etc.

Reply:
The idea was to expose both the 32 and 64 versions, but @reduz likely missed those here.